my-menu-tsps-web/src/main/webapp/css/my-menu-tsps.css

```diff
@@ -0,0 +1,34 @@
+/**
+ * Some custom CSS
+ **/
+
+.dashboard-settings.my-menu-tsps {
+  padding: 0;
+}
+
+.my-menu-tsps.col-sm-12 {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.my-menu-tsps .device-table {
+  padding: 15px;
+  margin-top: 0.5em;
+}
+
+/* Adjust the size of the columns in the table */
+.my-menu-tsps .column-monitor {
+  width: 30%;
+  max-width: 300px;
+  min-width: 80px;
+  font-family: Verdana, Arial, sans-serif;
+}
+
+#my-menu-dashboard {
+  color: rgb(178, 178, 178);
+  padding-left: 10px;
+  padding-top: 5px;
+  font-size: 3em;
+  font-family: OpenSans;

```

Comment:
Same here. I'd prefer to encourage future devs to use existing classes and stuff :-)